swarm/network/stream/lightnode_test.go

```diff
@@ -0,0 +1,187 @@
+package stream
+
+import (
+	"testing"
+
+	p2ptest "github.com/ethereum/go-ethereum/p2p/testing"
+)
+
+func TestLigthnodeRetrieveRequestWithRetrieve(t *testing.T) {
```

Comment:
please describe the tests in this file with comments before the 4 test functions

Reply:
Added comments